data/iriteshmishra.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "Ritesh Mishra",
+  "type": "personal",
+  "bio": "I am a Full Stack Developer",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://www.twitter.com/iRiteshMishra",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Youtube",
+      "url": "https://www.youtube.com/@iriteshmishra",
+      "icon": "FaYoutube"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/riteshmishra2",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/iriteshmishra",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/iriteshmishra",
+      "icon": "SiInstagram"
+    },
+    {
+      "name": "Email-Id",
+      "url": "iriteshmishra@gmail.com",
```

It should probably be `mailto:<mailID>`

```suggestion
      "url": "mailto:iriteshmishra@gmail.com",
```